package.json

```diff
@@ -48,9 +48,6 @@
 		"install-global-npm": "npm install -g npm@7.24.2",
 		"postinstall": "patch-package"
```

Comment:
We need to include the `patch-package` config + handling here as well, no?

Reply:
We don't seem to have any config for `patch-package`, it just applies the patches in `{repo root}/patches`.

Actually, we do have a minor hiccup there in that we have a PHP patch in that directory that that `patch-package` complains about, it's just a warning but we should fix it.